cmd/drone-docker/main.go

```diff
@@ -131,6 +131,11 @@ func main() {
 			Usage:  "build args",
 			EnvVar: "PLUGIN_BUILD_ARGS_FROM_ENV",
 		},
+		cli.StringSliceFlag{
+			Name:   "cache-from",
+			Usage:  "cache from",
```

Comment:
`images to consider as cache sources` would be better :P https://docs.docker.com/engine/reference/commandline/build/